Bottom: a6c8e32c3a9eed7e931da24ca5ef64ecce663c8e
Top:    99ea0a9e29839f35d4bbb22eb956babf8423b04c
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-04-26 19:19:28 +0100

drc: HACK HACK HACK


---

diff --git a/src/Makefile.am b/src/Makefile.am
index 4943755..353160a 100644
--- a/src/Makefile.am
+++ b/src/Makefile.am
@@ -285,6 +285,7 @@ libgtk_a_CPPFLAGS = -I./hid/gtk
 LIBGTK_SRCS = \
 	dolists.h \
 	hid/hidint.h \
+	hid/gtk/bga-wizard.c \
 	hid/gtk/gtkhid-main.c \
 	hid/gtk/gtkhid.h \
 	hid/gtk/gui.h \
